app/controllers/users_controller.rb

```diff
@@ -0,0 +1,29 @@
+class UsersController < ApplicationController
+  def show
+    @user = User.find params[:id]
+  rescue ActiveRecord::RecordNotFound
```

Comment:
nên viết ở application controller để bắt tất cả exception record not found em nhé